docs/core/linux-prerequisites.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Prerequisites for .NET Core on Linux
+description: Supported Linux versions and .NET Core dependencies to develop, deploy, and run .NET Core applications on Linux machines.
+keywords: .NET, .NET Core, Linux, debian, ubuntu, RHEL, centOS,
+author: johalex
+ms.author: johalex
+ms.date: 08/14/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
+---
+
+# Prerequisites for .NET Core on Linux
+
+This article shows the dependencies needed to develop .NET Core applications on Linux. The supported Linux distributions/versions, and dependencies that follow apply to the two ways of developing .NET Core apps on Linux:
+
+* [Command line](tutorials/using-with-xplat-cli.md)
+* [Visual Studio Code](https://code.visualstudio.com/)
+
+## Supported Linux versions
+
+.NET Core 2.x is supported on the following Linux distributions/versions:
```

Comment:
here since the versions are actually different, they can be placed in tabs if you want